client/db/src/bench.rs

```diff
@@ -132,7 +135,13 @@ impl<B: BlockT> StateBackend<HashFor<B>> for BenchmarkingState<B> {
 	type TrieBackendStorage = <DbState<B> as StateBackend<HashFor<B>>>::TrieBackendStorage;
 
 	fn storage(&self, key: &[u8]) -> Result<Option<Vec<u8>>, Self::Error> {
-		self.state.borrow().as_ref().ok_or_else(state_err)?.storage(key)
+		let state = self.state.borrow_mut();
+		let db_state = state.as_ref().ok_or_else(state_err)?;
+		let _ = self.usage_info.try_borrow_mut().map(|mut usage_info| {
```

Comment:
Anti pattern... I would like to try_borrow_mut, and if it fails simply "silence" it out, as the usage_info is not primordial

Reply:
You have single threaded code and you can just use `borrow_mut()`. If that fails, it means your code is broken and somewhere keeps a mutable reference. Please remove the `try_borrow_mut`